src/ert/parsing/lark_parser.py

```diff
@@ -116,7 +117,7 @@ class FileContextTransformer(Transformer):
     """Adds filename to each token,
     to ensure we have enough context for error messages"""
 
-    def __init__(self, filename):
+    def __init__(self, filename: str) -> "FileContextTransformer":
```

Comment:
Check liniting here, return type if `__init__` should be `None`.

Reply:
Done